util/test/start_test.py

```diff
@@ -126,7 +126,10 @@ def run_tests(tests):
     # futures or notests
     os.environ["CHPL_TEST_FUTURES"] = "1"
     os.environ["CHPL_TEST_NOTESTS"] = "1"
-    os.environ["CHPL_TEST_SINGLES"] = "1"
+    if args.respect_skipifs:
```

Comment:
A comment on top of this if can also be useful. Something like "if the user throws `--respect-skipifs`, we want to be able to skip even if it is a single test" maybe